pkgs/by-name/en/ente-auth/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ente-auth";
+  version = "3.1.1";
+
+  src = fetchurl {
+    url = "https://github.com/ente-io/ente/releases/download/auth-v${version}/ente-auth-v${version}-x86_64.AppImage";
+    hash = "sha256-WtFoqW++U+BoDUW1lc6xMJDCC1r2+O/fci492P2iVDg=";
```

Comment:
error: hash mismatch in fixed-output derivation '/nix/store/xvkjfr0bw2322p2bznw772n9y0c3wn8d-ente-auth-v3.1.1-x86_64.AppImage.drv':
         specified: sha256-WtFoqW++U+BoDUW1lc6xMJDCC1r2+O/fci492P2iVDg=
            got:    sha256-RcH+1YM6gPneVbv158LejClkrCNnAwr6K5hKbcyvTMI=